src/main/webapp/app/shared/layouts/navbar/navbar.component.ts

```diff
@@ -474,9 +476,6 @@ export class NavbarComponent implements OnInit, OnDestroy {
                 // This has to go in the future
                 this.addTranslationAsCrumb(currentPath, 'import');
                 break;
-            case 'import-from-file':
-                this.addTranslationAsCrumb(currentPath, 'import-from-file');
-                break;
```

Comment:
What does this have to do with this PR?

Reply:
nothing that's unintended. I guess related to my latest merge of develop.